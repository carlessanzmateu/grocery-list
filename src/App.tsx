import React, { useState }  from 'react';

import AppRouter from './common/app-router/appRouter';

function App() {
  return (
    <AppRouter />
  );
}

export default App;
