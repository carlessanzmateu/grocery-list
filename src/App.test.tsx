import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  test('should create', () => {
    const component = renderer.create(<App />);
    expect(component).toBeTruthy();
  });
  
  test('should match snapshot', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
})
