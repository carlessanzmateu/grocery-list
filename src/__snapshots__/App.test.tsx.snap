// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  <nav>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/favorites"
          onClick={[Function]}
        >
          Favorites
        </a>
      </li>
    </ul>
  </nav>
  <section
    className="grocery-list-view"
  >
    <div
      className="list"
    >
      <div
        className="vertical-card-list"
      >
        <header
          className="header"
        >
          List
        </header>
        <div
          className="cart-summary"
          onClick={[Function]}
        >
          <strong>
            See my cart
             (
            0
            )
          </strong>
        </div>
        <div
          className="content-wrapper"
        />
      </div>
    </div>
    <div
      className="cart"
    >
      <div
        className="horizontal-card-list"
      >
        <div
          className="summary-zone"
        >
          <header
            className="header"
          >
            Cart
          </header>
          <div
            className="cart-summary"
            onClick={[Function]}
          >
            <strong>
              Go back
               (
              0
              )
            </strong>
          </div>
          <div
            className="content-wrapper"
          />
        </div>
        <div
          className="interaction-zone"
        >
          <p
            className="summary"
          >
            Total amount: 
            0
            $
          </p>
          <a
            className="button"
            onClick={[Function]}
          >
            <strong>
              Checkout
            </strong>
          </a>
        </div>
      </div>
    </div>
  </section>
</div>
`;
